nixos/modules/services/security/vaultwarden/default.nix

```diff
@@ -62,20 +62,70 @@ in {
       default = {};
       example = literalExpression ''
         {
-          domain = "https://bw.domain.tld:8443";
-          signupsAllowed = true;
-          rocketPort = 8222;
-          rocketLog = "critical";
+          enable = true;
+          dbBackend = "sqlite";
+          config = {
```

Comment:
I think an example should only show a possible value for its own option, so here that would be only the attrset inside `config = { ... }`, without `enable`, `dbBackend` etc, which are documented where they're defined.

Reply:
You are right. Fixed.